### ISSUE TITLE
yum update may fail when jubadump is previously installed 

### ISSUE THREAD

**Issue by kmaehashi** [jubatus/jubatus#1060]
Currently jubadump package declares explicit version dependency (introduced in #1001) to jubatus-core. However, as jubadump may not be updated when we update jubatus-core.

So, this causes RPM update failure in the following scenario:

1. (After Jubatus 0.8.3 + jubadump 0.8.0 release) `yum install jubadump`
1. (After Jubatus 0.8.4 + jubadump 0.8.0 release) `yum update`

The error message is:

```
Error: Package: jubadump-0.8.0-1.el6.x86_64 (@jubatus)
           Requires: jubatus-core = 0.2.3
           Removing: jubatus-core-0.2.3-1.el6.x86_64 (@jubatus)
               jubatus-core = 0.2.3-1.el6
           Updated By: jubatus-core-0.2.4-1.el6.x86_64 (jubatus)
               jubatus-core = 0.2.4-1.el6
```

For jubadump pacakge, we should declare version requirement as `jubatus-core >= @JUBATUS_CORE_VERSION@` instead of `jubatus-core = @JUBATUS_CORE_VERSION@`.

**Comment by kmaehashi** [comment]
If you face this error, you can workaround the problem by reinstalling jubadump package.

```
yum remove jubadump
yum install jubadump
```

This issue only affects to users who installed jubadump via RPM.

**Comment by kmaehashi** [comment]
Fixed via #1062
